test/fixtures/tree-shaking/src/main.js

```diff
@@ -10,8 +10,17 @@ import readline from 'readline';
 import repl from 'repl';
 import tls from 'tls';
 
+import defaultExportBrowserPlugin from './plugins/default-export-browser.js';
+import defaultExportNodePlugin from './plugins/default-export-node.js';
+import instrumentedAsPureBrowserPlugin from './plugins/instrumented-as-pure-browser.js';
+import instrumentedAsPureNodePlugin from './plugins/instrumented-as-pure-node.js';
+
 export default async function() {
   const app = new App('element', el => el);
+  __BROWSER__ && app.register(defaultExportBrowserPlugin);
```

Comment:
I think it would be really useful to test named exports also.

Reply:
Good idea, test added.